segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java

```diff
@@ -328,8 +361,7 @@ private void handleException(UUID writerId, long requestId, String segment, long
             connection.close();
         } else if (u instanceof TokenExpiredException) {
             log.warn(requestId, "Token expired for writer {} on segment {}.", writerId, segment, u);
```

Comment:
Should this be bumped to ERROR log level ?

Reply:
`ERROR` would be too harsh here, as the system continues to function normally.